### PR TITLE
Feature/footerdesign

### PR DESCRIPTION
**Pull request by olmeiro**
This is the first approach to designing the footer component. 
I used the next libraries to take advantage:
https://windstatic.com/
https://ionic.io/ionicons/usage

Let me know what you think.

### CODE REVIEW

File: src/components/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        >
+          <div className="px-5 py-2">
+            <a href="#" className="text-sm text-gray-500 hover:text-blue-600">
+              Follow us
+            </a>
+          </div>
+        </nav>
+        <div className="flex justify-center mt-4 space-x-6">
+          <span className="inline-flex justify-center w-full gap-3 m-auto md:justify-start md:w-auto">
+            <Link
```
Comment (dafegosa) [inline src/components/Footer.jsx]: # Advice

You have 4 different Links that have very similar content. So, you can create an array with the links information, for example:

```javascript
const SOCIAL_NETWORKS = [
  {
    name: 'github',
    Label: 'Github',
    url: ''
  },
  {
    name: 'twitter',
    Label: 'Twitter',
    url: ''
  },
  {
    name: 'instagram',
    Label: 'Instagram',
    url: ''
  },
  {
    name: 'linkedin',
    Label: 'LinkedIn',
    url: ''
  },
]
```

And render the Links container by iterating the array:
```javascript
{ SOCIAL_NETWORKS.map(socialNetwork => {
return (
    <Link
        to={socialNetwork.url}
        target="_blank"
        className="w-6 h-6 transition fill-black hover:text-blue-500"
      >
           <span className="sr-only">{socialNetwork.Label}</span>
           <ion-icon
                className="w-5 h-5 md hydrated"
                name=`logo-${socialNetwork.name}`
                role="img"
                aria-label=`logo ${socialNetwork.name}`
            ></ion-icon>
      </Link>
)})
}

**Review by dafegosa — CHANGES_REQUESTED** [review]

**Comment by dafegosa** [comment]
@olmeiro, related to the libraries you are using: I have never used them. Let's try them 👍

**Comment by olmeiro** [comment]
I just changed the code as you suggested, thanks for sharing your knowledge. I hope works well

**Review by dafegosa — APPROVED** [review]
Good job!

**Pull request merged**